app/backend/src/couchers/languages.py

```diff
@@ -0,0 +1,19 @@
+import functools
+import json
+from pathlib import Path
+
+
+@functools.cache
+def get_language_list():
+    """
+    Get list of allowed languages as a set.
+    """
+    with open(Path(__file__).parent / "constant_data" / "languages.json", "r") as f:
+        return set(language["code"] for language in json.loads(f.read()))
```

Comment:
Can be written shorter as `json.load(f)`.

Reply:
The annoyance with `.load` is that it won't accept a `pathlib.Path`, and it feels somehow ugly to do a `str` on a `Path`

Reply:
it takes an already opened file handle.

Reply:
Ah right, yeah, makes sense